packages/field/src/FormatMixin.js

```diff
@@ -336,14 +336,19 @@ export const FormatMixin = dedupeMixin(
           // is guaranteed to be calculated
           setTimeout(this._reflectBackFormattedValueToUser);
         };
```

Comment:
this is only adding a "guard" to the `this.inputElement`... so the FormatMixin also works when applied to classes that do not have an underlying native inputElement...

=> should be extracted into a separate MR